### PR TITLE
set kubernetes version to 1.12.8 to match current release

### PR DESCRIPTION
**Pull request by chrisz100**

### CODE REVIEW

File: channels/alpha [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
     recommendedVersion: 1.13.5
     requiredVersion: 1.13.0
   - range: ">=1.12.0"
-    recommendedVersion: 1.12.7
+    recommendedVersion: 1.12.8
```
Comment (mikesplain) [inline channels/alpha]: Thanks Chris! You actually need to modify the file down below as well:

https://github.com/kubernetes/kops/blob/52e9237d7240806d38d11e277af4a19dbe779006/channels/alpha#L81

File: channels/stable [hunk 2]
```diff
@@ -78,7 +78,7 @@ spec:
   - range: ">=1.12.0-alpha.1"
     #recommendedVersion: "1.12.0"
     #requiredVersion: 1.12.0
-    kubernetesVersion: 1.12.7
+    kubernetesVersion: 1.12.8
```
Comment (mikesplain) [inline channels/stable]: We generally hold off on bumping stable right away unless there are security vulnerabilities. Once a few people, attest they've been running it successfully in prod we bump. Do you feel like we can hold off on modifying stable for a bit?

File: channels/alpha (hunk 1 above)
Comment (chrisz100) [reply channels/alpha]: Damn I knew there was another file! On the way

File: channels/stable (hunk 2 above)
Comment (chrisz100) [reply channels/stable]: As we are not on a stable release yet I don't really see a reason to hold this off much longer tbh. But I can keep this out for now as well

**Comment by mikesplain** [comment]
Thanks @chrisz100!

/lgtm
/approve

**Pull request merged**